src/Observable.ts

```diff
@@ -89,7 +90,7 @@ export class Observable<T> implements CoreOperators<T>  {
     let subscriber: Subscriber<T>;
 
     if (observerOrNext && typeof observerOrNext === 'object') {
-      if (observerOrNext instanceof Subscriber || observerOrNext instanceof Subject) {
+      if (observerOrNext instanceof Subscriber) { //|| observerOrNext instanceof Subject) {
```

Comment:
Yes, need to cast `<any>` instead of `Subject` (since compile time imported ambient declaration does not have interfaces) but this works, thanks for pointing out. I was just spending time in opposite way of bringing interface check availability to runtime..